app/src/main/java/org/p2p/wallet/home/ui/main/empty/adapterdelegates/emptyHomeBannerAdapterDelegate.kt

```diff
@@ -0,0 +1,30 @@
+package org.p2p.wallet.home.ui.main.empty.adapterdelegates
+
+import com.hannesdorfmann.adapterdelegates4.AdapterDelegate
+import com.hannesdorfmann.adapterdelegates4.dsl.adapterDelegateViewBinding
+import org.p2p.wallet.databinding.ItemBigBannerBinding
+import org.p2p.wallet.home.model.EmptyHomeItem
+import org.p2p.wallet.home.model.HomeBannerItem
+import org.p2p.wallet.utils.viewbinding.inflateViewBinding
+
+fun emptyHomeBannerAdapterDelegate(
```

Comment:
AdapterDelegates creates delegates with the function call like this
I haven't used it before, but its just an example what can we use